components/filter-components/Timeline.vue

```diff
@@ -1,21 +1,24 @@
 <template>
-  <!--<div class="h-40 bg-slate-300 flex items-center
-             justify-center z-20"
-       style="width: 80%;">-->
-
-  <div class="bg-slate-200 z-30 py-5 px-12 filters-shadow
-              rounded-tr-xl rounded-br-xl"
-       style="min-height: 10%; max-height: 17%; width: 95%; position: absolute; bottom: 0%;">
-    <vue-slider class=""
-                v-model="rangeValue"
-                :data="dates"
-                :data-value="'date'"
-                :data-label="'label'"
-                :marks="markers">
-    </vue-slider>
+  <div v-if="timelineShowing">
```

Comment:
This may be super tricky to do, but when you close this timeline, it would be nice if the filter bar expands to fill the space. Hope this makes sense